one_fm/hiring/doctype/work_contract/work_contract.py

```diff
@@ -65,6 +70,21 @@ def on_cancel(self):
 	def on_trash(self):
 		if self.docstatus == 0:
 			update_onboarding_doc(self, True)
+	
+	def check_for_applicant_signature(self):
+		if self.employee_signature:
+			return True
+		else:
+			return False
+	
+	def set_authorized_signatory(self):
+		#This function fetch the Authorized Signatory and sets "authorised_signatory_signature" field.
+		# for Time being, it's not being called. 
+		signature_doc_list = frappe.get_doc("PIFSS Authorized Signatory", {'name':"شركة ون لإدارة المرافق"})
+		for doc_list in signature_doc_list.authorized_signatory:
+			if doc_list.authorized_signatory_name_english == "Abdullah":
```

Comment:
Why hard coded value here?
Can't this authorized signatory be set in any doctype?